### PR TITLE
Feat/update feedback component

### PR DESCRIPTION
**Pull request by mariaozamiz**
### :pushpin: References

-   **Issue:** Closes [Feedback tools is not able to upload screenshots](https://app.clickup.com/t/8693egcc4)

### :memo: Implementation

Replace old feedback-tool with feedback-component. 

### :video_camera: Screenshots/Screen capture
Captura de pantalla 2024-05-07 a las 12 45 40

### :fire: Testing

Launch the project and submit your feedback using the feedback component. To ensure the action was successful, navigate to the network tab. Once confirmed, head over to the [ClickUp list](https://app.clickup.com/4528615/v/l/6-44406062-1) and find there your submitted feedback.

### CODE REVIEW

File: src/webapp/pages/app/App.tsx [hunk 1]
```diff
@@ -53,14 +54,13 @@ export const App: React.FC<AppProps> = React.memo(function App({ api, d2, instan
                 <SnackbarProvider>
                     <LoadingProvider>
                         <HeaderBar appName="User Extended App" />
-
                         <div id="app" className="content">
                             <AppContext.Provider value={appContext}>
                                 <Router />
                             </AppContext.Provider>
                         </div>
-
```
Comment (p3rcypj) [inline src/webapp/pages/app/App.tsx:62]: This is subjective but adding blank lines can improve readability for blocks that have multiline components. Another approach would be to abstract the `div#app` into another component so we will have 4 components of one line. Based on [3DBIONOTES PR #214](https://github.com/EyeSeeTea/3DBIONOTES/pull/214#discussion_r1567069654).

File: src/webapp/pages/app/AppConfig.ts [hunk 2]
```diff
@@ -1,21 +1,11 @@
...
-            title: string;
-            body: string;
-        };
-        snapshots: {
-            repository: string;
-            branch: string;
-        };
-        feedbackOptions: object;
+    app: {
+        notifyEmailOnProjectSave: string[];
```
Comment (p3rcypj) [inline src/webapp/pages/app/AppConfig.ts:8]: As we discussed, to be removed

File: src/app-config.ts [hunk 3]
```diff
@@ -1,43 +1,24 @@
+import { AppConfig } from "./webapp/pages/app/AppConfig";
+
 export const appConfig: AppConfig = {
     appKey: "user-extended-app",
     appearance: {
         showShareButton: false,
     },
+    app: {
+        notifyEmailOnProjectSave: [""],
```
Comment (p3rcypj) [inline src/app-config.ts:9]: To be removed

**Review by p3rcypj — CHANGES_REQUESTED** [review]
Thank you, @mariaozamiz, for adding the new component 🙌🏻 Everything looks good! There are just some minor things to be removed but everything is ok and the functionality works perfectly👌🏻

**Review by p3rcypj — APPROVED** [review]
Everything looking good! Thanks!! @mariaozamiz

**Review by MiquelAdell — APPROVED** [review]
working fine, thanks!

**Pull request merged**